src/config/settings.rs

```diff
@@ -155,7 +155,7 @@ impl FromStr for Setting {
 
     /// Converts a string to a setting or None when the string is invalid
     fn from_str(key: &str) -> Result<Setting> {
-        let (key_type, key_value) = key.split_once(':').unwrap();
+        let (key_type, key_value) = key.split_once(':').expect("");
```

Comment:
something that says, what has failed in the expect would be nice, else it is more or less the same as `unwrap`